### ISSUE TITLE
Apostrophes are escaped in wager chat

### ISSUE THREAD

**Issue by Dan-McL** [Dan-McL/Challenge-Master#9]
This reduces readability and make chatting harder so need to stop escaping ' or make it so that it displays the HTML code properly.

**Comment by DeadRed6** [comment]
Unlikely, but I wonder if this built-in function interferes with your coded one of the same name:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape

**Comment by Dan-McL** [comment]
I don't think it is because that method escapes `'` to `%27` whereas in chat it is `&#x27;`
